pkg/webservice/handlers.go

```diff
@@ -418,6 +420,80 @@ func getNodesUtilJSON(partition *scheduler.PartitionContext, name string) *dao.N
 	}
 }
 
+func getNodeUtilisations(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+	var result []*dao.PartitionNodesUtilDAOInfo
+	for _, part := range schedulerContext.GetPartitionMapClone() {
+		partitionNodesUtilJSON := getPartitionNodesUtilJSON(part)
```

Comment:
```go
result = append(result, getPartitionNodesUtilJSON(part))
```

how about this style?

Reply:
Fixed in the latest commit 0b66a545850ace2d724d7db284f3083462a87dbe